packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -133,6 +133,16 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
             },
             mouseout: () => {
                 this.props.tooltipComponent?.setElement(undefined);
+            },
+            click: (el, ev, clickCount) => {
+                if (clickCount === 2) {
+                    const start = el.model.range.start;
+                    const end = el.model.range.end;
+                    this.props.unitController.viewRange = {
```

Comment:
Some time chart elements have zero duration range (start == end) like annotations. We should not attempt to set the viewRange in that case. Although internally, the unit controller will not update the view range but it will still broadcast a viewRangeChanged event with the old view range.